gwvet/tabs.py

```diff
@@ -244,13 +254,15 @@ def init_plots(self, plotdir=os.curdir):
                     'ylabel': params.get(
                         'frequency-label',
                         get_column_label(params['frequency'])),
+                    'ylim': params.get('ylim', [10,5000]),
```

Comment:
@tjma12, can you include a space separating the elements?

```suggestion
                    'ylim': params.get('ylim', [10, 5000]),
```